packages/semver/src/schematics/ng-add/utils/dependencies.ts

```diff
@@ -0,0 +1,164 @@
+import { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+
+import { SchemaOptions } from '../schema';
+import {
+  getMergedPackageJsonConfig,
+  getPackageJson,
+  overwritePackageJson,
+  PackageJson,
+  PackageJsonConfigPart,
+} from './package-json';
+
+const COMMITIZEN = {
+  commitizen: '^4.2.4',
+  'cz-conventional-changelog': '^3.3.0',
+};
+
+const COMMITLINT = {
+  '@commitlint/cli': '^12.1.4',
+  '@commitlint/config-conventional': '^12.1.4',
+};
+
+const HUSKY = {
+  husky: '^6.0.0',
+};

```

Comment:
Can we move this into a dedicated `packages/semver/src/schematics/ng-add/dependencies.ts` file?

Reply:
I agree... done